lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -433,11 +440,39 @@ after insert => sub {
         meta2        => join('', map { sprintf ':%s:', $_ }
             $self->topics->columns([qw/ name /])->all),
         description  => sprintf('Instant Answer Page [%s](%s) created!',
-            $self->name, sprintf('https://duck.co/ia/view/%s',
-                $self->meta_id)),
+            $self->name, $self->uri( { activity_feed => 1 } ) ),
     } );
 };
 
+sub create_update_activity {
+    my ( $self, $meta3, $description ) = @_;
+    $self->result_source->schema->resultset('ActivityFeed')->updated_ia(
+        meta1        => $self->id,
+        meta2        => $self->topics->join_for_activity_meta( 'name' ),
+        meta3        => $meta3,
+        description  => $description,
+    );
+}
+
+sub _generate_updates {
+    my ( $self, $update ) = @_;
+
+    while ( my ($column, $value) = each $update ) {
+
+        if ( $column eq 'dev_milestone' ) {
```

Comment:
The most important  (excluding dev milestone) are:
- name
- meta_id (it can change since it's editable)
- type
- perl module

For the purpose of this PR I think we're good with just dev milestone. Maybe meta_id too since it decides the URL